### PR TITLE
docs: Two little improvements regarding `get_data` documentation

### PR DESCRIPTION
**Pull request by ldoktor**
Just a 2 notes that might improve the documentation regarding recently added `get_data`.

### CODE REVIEW

File: docs/source/WritingTests.rst [hunk 1]
```diff
@@ -213,6 +213,9 @@ Avocado looks for data files in the order defined at
 from most specific one, to most generic one.  That means that, if a variant
 is being used, the **variant** directory is used first.  Then the **test**
 level directory is attempted, and finally the **file** level directory.
+Additionally you can use ``get_data(filename, must_exist=False)`` to get
```
Comment (clebergnu) [inline docs/source/WritingTests.rst]: Yep, fair enough.

File: docs/source/WritingTests.rst [hunk 2]
```diff
@@ -208,6 +208,10 @@ to be accessed from up to three sources:
    ``MyTest.test_foo``, with variant ``debug-ffff``, the data directory path
    will be ``/home/user/test.py.data/MyTest.test_foo/debug-ffff/``.
 
+.. note:: Unlike INSTRUMENTED tests the ``SIMPLE`` tests only define `file`
```
Comment (clebergnu) [inline docs/source/WritingTests.rst]: There's an inconsistency here in how you format test types (without double back quotes for INSTRUMENTED, and with them for SIMPLE.

Comment (clebergnu) [reply docs/source/WritingTests.rst]: Also, it sounds strange to me (remember, not native English speaker) to use "the SIMPLE tests", when the previous test referenced (INSTRUMENTED) did not have an definite article preceding them.  I'd write that as: "Unlike INSTRUMENTED tests, SIMPLE tests only define..."

File: docs/source/WritingTests.rst [hunk 3]
```diff
@@ -208,6 +208,10 @@ to be accessed from up to three sources:
...
+          and `variant` data_dirs, therefor the most-specific data-dir
```
Comment (clebergnu) [inline docs/source/WritingTests.rst]: Final note here is that I think "therefore" is the word you were looking. Therefor is a valid word, but AFAICT, has a slight different meaning: https://www.grammarly.com/blog/therefore-vs-therefor/

File: docs/source/WritingTests.rst (hunk 2 above)
Comment (ldoktor) [reply docs/source/WritingTests.rst]: Yep, looks better...

File: docs/source/WritingTests.rst (hunk 3 above)
Comment (ldoktor) [reply docs/source/WritingTests.rst]: yep, sorry, it's because I learned English from movies and I suck at written form...

**Pull request closed without merging**